.editorconfig

```diff
@@ -9,7 +9,7 @@ root = true
 
 # Change these settings to your own preference
 indent_style = space
-indent_size = 4
+indent_size = 2
 
```

Comment:
Editors tend to pick what's around, so it might be good idea to do one massive PR instead of leaving mixed styles behind.